pkgs/applications/version-management/sr-ht/core.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, fetchgit, fetchNodeModules
+, nodejs-11_x, python3Packages
+, sassc }:
+
+with python3Packages;
+
+buildPythonPackage rec {
+  name = "core-sr-ht-${version}";
+  version = "0.30.5";
+
+  src = fetchgit {
+    url = "https://git.sr.ht/~sircmpwn/core.sr.ht";
+    rev = version;
+    sha256 = "0ccbymmixldf0mb8dmcij2j7v0cxp1vhrfbl6az1hv5y1rxijqb6";
+  };
+
+  node_modules = fetchNodeModules {
+    src = "${src}/srht";
+    nodejs = nodejs-11_x;
+    sha256 = "0axl50swhcw8llq8z2icwr4nkr5qsw2riih0a040f9wx4xiw4p6p";
+  };
+
+  patches = [
+    ./npm.patch
```

Comment:
What about this one? The files should have more descriptive names.

Reply:
`npm.patch` is suppose to remove the code that calls `npm i` since the derivation doesn't have network access, `node_modules` are fetched through `fetchNodeModules`.